stanford/roles/cluster/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 CLUSTER_AMI: 'ami-07585467'
 CLUSTER_ASSIGN_PUBLIC_IP: no
+CLUSTER_DB_INSTANCE_SIZE: 5
+CLUSTER_DB_INSTANCE_TYPE: db.t2.micro
```

Comment:
We probably don't need a new variable for this, but can keep using `CLUSTER_INSTANCE_TYPE`,
since it only ever runs _either_ `ec2` or `rds` in a single invocation.